upgrade/1.0.11/Stage_0_Prerequisites.md

```diff
@@ -0,0 +1,181 @@
+# Stage 0 - Prerequisites and Preflight Checks
+
+> **NOTE:** CSM-1.0.1 or later is required in order to upgrade to CSM-1.0.11.
+>
+> **NOTE:** Installed CSM versions may be listed from the product catalog using the following command. This will sort a semantic version without a hyphenated suffix after the same semantic version with a hyphenated suffix, e.g. 1.0.11 > 1.0.11-beta.19.
+>
+
+Use the following command can be used to check the CSM version on the system:
+
+```bash
+ncn# kubectl get cm -n services cray-product-catalog -o json | jq -r '.data.csm'
+```
+
+This check will also be conducted in the 'prerequisites.sh' script listed below and will fail if the system is not running CSM-1.0.1 or CSM-1.0.10.
+
+>**`IMPORTANT:`**
+> 
+> Before running any upgrade scripts, be sure the Cray CLI output format is reset to default by running the following command:
+>
+>```bash
+> ncn# unset CRAY_FORMAT
+>```
+
+## Stage 0.1 - Install latest docs RPM
+
+1. Install latest document RPM package:
+
+    * Internet Connected
+
+        ```bash
+        ncn-m001# cd /root/
+        ncn-m001# wget https://storage.googleapis.com/csm-release-public/shasta-1.5/docs-csm/docs-csm-latest.noarch.rpm
+        ncn-m001# rpm -Uvh docs-csm-latest.noarch.rpm
+        ```
+
+    * Air Gapped
+
+        ```bash
+        ncn-m001# cd /root/
+        ncn-m001# rpm -Uvh [PATH_TO_docs-csm-*.noarch.rpm]
```

Comment:
Is the text in [] supposed to be replaced?

Reply:
adding comment